lib/pages/people-page.js

```diff
@@ -57,18 +57,38 @@ export default class PeoplePage extends AsyncBaseContainer {
 	}
 
 	async viewerDisplayed( username ) {
-		return DriverHelper.isEventuallyPresentAndDisplayed(
-			this.driver,
-			By.css( `.people-profile__login[data-e2e-login="${ username }"]` )
+		await DriverHelper.isEventuallyPresentAndDisplayed( this.driver, By.css( '.count' ) );
+		let noOfViewers = parseInt(
+			await this.driver.findElement( By.css( '.section-header__label .count' ) ).getText()
 		);
+
+		let retries = Math.round( noOfViewers / 10 );
+		let displayed;
+
+		for ( let i = 0; i <= retries; i++ ) {
```

Comment:
List will be scrolled enough times to check every user. `retries` is rounded number of all users divided by 10.